app/src/main/res/values/strings.xml

```diff
@@ -18,4 +18,23 @@
     <string name="rate">RATE APP</string>
```

Comment:
Yep, but i don't have the `textAllCaps` because in the navigation view, in which I use it, I don't have that property.